server/src/schemas/equipmentSchema.ts

```diff
@@ -27,6 +27,6 @@ export const EquipmentTypeDefs = gql`
   extend type Mutation {
     addEquipment(equipment: EquipmentInput): Equipment
     updateEquipment(id: ID!, equipment: EquipmentInput): Equipment
-    removeEquipment(id: ID!): Equipment
+    archiveEquipment(id: ID!): Equipment
```

Comment:
Since the concept of archiving things is just an implementation detail, I think it should be `deleteEquipment`. If I were reading the API for the first time and encountered `archivedEquipment`, I would assume that users are able to archive & unarchive stuff, when in reality it is a permanent deletion (from the users perspective). lmk if that makes sense

Reply:
I see it go both ways, atm been following archive naming format similar how users where done. If this wants to be changed to be deleteUsers() or deleteEquipment() as the naming format, it can be an easy name change.